### ISSUE TITLE
Add Python Action

### ISSUE THREAD

**Issue by huangwc94** [muesli/beehive#176]
Hi, This is a fantastic go project!
Just curious if we can get a python action that can execute customize python code, this will extends the project with much much more functionalities that suit a wide range of user. 
Also, other scripting language is useful as well.

**Comment by alkuzad** [comment]
yeah this is something I came here looking for alternatives to big old Huginn
@muesli is this project still accepting PRs or is it abandoned ?

**Comment by muesli** [comment]
@alkuzad: Active and accepting PRs.

**Comment by sabhiram** [comment]
How about just using the `execbee` to run `python`? I would have called it `shelbee` for the lulz, but awesome project!

**Comment by Zykino** [comment]
If this is about executing python program with beehive, the "execute command" hive should be enough. If it is about writting an hive in Python/scripting language, well I don’t know Go but if you could expose C bindings it could let peoples write hives in any language.
